README.markdown

```diff
@@ -5639,7 +5639,7 @@ Returns a read-only cosocket object that wraps the downstream connection. Only [
 
 In case of error, `nil` will be returned as well as a string describing the error.
 
-Due to the streaming nature of HTTP2 and HTTP3, this API cannot be used when the downstream connection is HTTP2 and HTTP3.
+Due to the streaming nature of HTTP2 and HTTP3, When the downstream connection maintains a continuous stream (such as GRPC), a block will occur. The block time depends on the [client_body_timeout](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_timeout) setting. After the timeout, all Lua codes will be skipped and a 408 status code will be returned. [#2286](https://github.com/openresty/lua-nginx-module/pull/2286)
```

Comment:
**Note:** This method will block while waiting for client request to be fully received. The block time depends on the [client_body_timeout](http://nginx.org/en/docs/http/ngx_http_core_module.html#client_body_timeout) directive and maximum body size specified by the [client_max_body_size](https://nginx.org/en/docs/http/ngx_http_core_module.html#client_max_body_size) directive. If read timeout occurs or client body size exceeds the defined limit, this function will not return and `408 Request Time-out` or `413 Request Entity Too Large` response will be returned to the client instead.

Reply:
changed!